scripts/generate-manifest.js

```diff
@@ -60,7 +60,7 @@ const devManifest = {
 const prodManifest = {
   name: 'Hiro Wallet',
   content_security_policy:
-    "script-src 'self'; object-src 'none'; frame-src 'none'; frame-ancestors 'none';",
+    "default-src 'none'; connect-src *; style-src 'unsafe-inline'; script-src 'self'; object-src 'none'; frame-src 'none'; frame-ancestors 'none';",
```

Comment:
@kyranjamie can you explain what this change does?

Reply:
Yes, previously we didn't have a default-src, which sets all srcs. This adds it, and enables the min we need `connect-src *` (to allow any node someone might add), and `src-src 'unsafe-inline'` how emotion works